extensions-jvm/smooks/runtime/pom.xml

```diff
@@ -42,6 +42,12 @@
         <dependency>
             <groupId>org.apache.camel</groupId>
             <artifactId>camel-smooks</artifactId>
+            <exclusions>
+                <exclusion>
+                    <groupId>org.smooks.cartridges.edi</groupId>
+                    <artifactId>smooks-edi-cartridge</artifactId>
+                </exclusion>
+            </exclusions>
```

Comment:
Please can we do this in the CQ BOM instead of directly in the extension.

Reply:
Thanks. updated